main.go

```diff
@@ -40,6 +45,18 @@ const (
 	mapiGroup = "machine.openshift.io"
 )
 
+var (
+	// Default leader electrion configuration.
+	leaderElectionConfig = config.LeaderElectionConfiguration{
+		LeaderElect:       true,
+		LeaseDuration:     metav1.Duration{Duration: 137 * time.Second},
```

Comment:
why did you choose this values?

Reply:
copied from this commit of cccmo: https://github.com/openshift/cluster-cloud-controller-manager-operator/commit/445bfe3e7d7e78dea80b0dec3f729881880f21db